jax_unirep/params.py

```diff
@@ -1,11 +1,10 @@
 from typing import Dict
 
-from jax.nn.initializers import glorot_normal
 from jax.random import PRNGKey, normal, split
 
 key = PRNGKey(42)
 
-
+# TODO: This whole module seems deprecated, maybe remove?
```

Comment:
Blessed is the developer who proposes to delete code!

Reply:
Let me run a check using [`pydeps`](https://github.com/thebjorn/pydeps).

Reply:
Just as you had suspected:

![image](https://user-images.githubusercontent.com/2631566/100805046-fbd85a00-33fb-11eb-8c9a-7037c5b137bf.png)

`params.py` is quite low-level, but isn't used anywhere inside the library. I think we can delete it safely.

Reply:
Nice tool!